### ISSUE TITLE
Funny filters

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/spreed#20]
»Must have feature« – confirmed by @karlitschek

**Comment by LukasReschke** [comment]
Ref https://auduno.github.io/clmtrackr/face_mask.html

**Comment by LukasReschke** [comment]
https://github.com/auduno/clmtrackr

**Comment by LukasReschke** [comment]
https://auduno.github.io/clmtrackr/examples/caricature.html

**Comment by jancborchardt** [comment]
Postponed to 1.1 :frowning:

**Comment by jancborchardt** [comment]
@nickvergessen can you elaborate on the closing?

While I understand that it sounds silly, every video chat app has some kind of funny smiley or filter function which people always like to use.

It's not a priority for sure, but also not something we should outright ignore.

**Comment by nickvergessen** [comment]
I didn't close it because it "sounds silly", but because there should be 0 energy wasted on this, while we are fighting much bigger/fundamental problems: reconnection, http2 support, quality negotiation, popup, ringing, ...

**Comment by jancborchardt** [comment]
@nickvergessen of course. That’s what prioritization is for. But the »not important yet« issues are not to be closed, but put in the backlog. ;)
